### DISCUSSION TITLE
Commercial Product License

### DISCUSSION THREAD

**Discussion by andrew-915** [InfinyTech3D/SofaUnity#96] [General topics regarding SofaUnity]
Hi there 
I have just tested the trial license of SOFA Unity project , Great work we like it. 
but I have a question before we can proceed with buying a commercial product license , 
in the pricing page "Number of site (License per computer) "  That mean each computer we are going to deploy our end user product will need a license or it just for developer computer which runs Unity .
We will be very appriated if this point is cleared and if there is any sales representative contact it will help a lot.
Thanks in advance.

**Comment by epernod** [comment]
Dear,

the [license page](https://infinytech3d.com/licenses/) has been updated with major change linked to the release of the asset on the Unity Store.
Let us know if it still unclear, we will make sure to answer you.

Best regards,

**Reply by epernod** [reply]
any update @andrew-915 
Thank you
